src/App.js

```diff
@@ -154,7 +155,7 @@ class App extends Component {
     render() {
         return (
             <Loop>
-                <Stage style={ { background: '#000' } }>
+                <Stage style={ { backgroundColor: '#000 ', backgroundImage: 'url(' + theme1Bkgd + ')',  backgroundSize: '180px 180px'  } }>
```

Comment:
We should probably export this to a stylesheet if it gets any lengthier.

Reply:
Agreed.  In a future iteration, I see us implementing Themes such that we could change the look (tile, apple color, scoreboard font color) of the game board if, say, all apples are eaten or something.  That would require changing the behaviour of the game-kit stuff were using so I didn't quite have the time to do that yet.